logic/common/log_utils.py

```diff
@@ -0,0 +1,112 @@
+''' Utility functions for printing and logging
+
+USAGE EXAMPLE:
+
+import logic.common.utils_log as log
+    log.SetVerbosityLevel(1)
+    log.Info("Hello World!")
```

Comment:
They are indented differently because the lines are used in different cases. More specifically, the importing is done at the beginning of file, while the initialisation is wraped within the main() function. I intended the usage lines to be copied and pasted onto the files directly as is. As in:

- Console:
`python cli_collide.py j01`
- CLI import:
`import argparse`
`import logic.common.utils_log as log`
`...`
- CLI main():
`main()`
`    args = parser.parse_args()`
`    log.SetVerbosityLevel(args.v)`
`    ...`

As you can see, the `import` line would not have indentation, whereas `SetVerbosityLevel()` is within the main function and requires indentation. So if I remove that in the USAGE section, I would have to add the extra space each time after copying - not the most convenient.


--------------------------------------------------

How about for the file description at the beginning of `cli_collide`, we do:

```
-USAGE- (Console Commands)
python cli_collide.py j01
python cli_collide.py j01 --v 2
```

And for `log_utils`, we do:

```
-USAGE- (Import)
import logic.common.utils_log as log

-USAGE- (Main)
    log.SetVerbosityLevel(1)
    log.Info("Hello World!")
```